package.json

```diff
@@ -8,14 +8,16 @@
     "deploy:nightlies": "gulp deploy_nightlies",
     "build": "gulp build && electron-builder -p never",
     "build:mas": "gulp build && electron-builder --config build-mas.yml",
+    "build:arm64": "gulp build && electron-builder -p never --arm64",
     "start": "electron .",
     "start_debug": "electron . --dev --inspect",
     "dev": "electron . --dev --inspect",
     "clean": "gulp clean",
     "lint:js": "eslint . --ext .es --ext .js --ext .ts --ext .tsx --ignore-path .gitignore",
     "lint:css": "stylelint views/**/*.css assets/**/*.css --config .stylelintrc.css.js",
     "lint:styled": "stylelint views/**/*.es --config .stylelintrc.js",
-    "lint": "npm run lint:js && npm run lint:css && npm run lint:styled"
+    "lint": "npm run lint:js && npm run lint:css && npm run lint:styled",
+    "postinstall": "patch-package"
```

Comment:
This npm script will run for each `npm install`

since we don't always build the app, there's no need to patch every time.I think we can call it before we start to patch